network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/partner.html

```diff
@@ -11,7 +11,7 @@ <h2 class="tw-h3-heading">{{ page.partner_heading }}</h2>
                         {{ page.partner_intro_text }}
                     </p>
                 {% endif %}
-                <a href="{{ page.partner_page.localized.url }}" class="text-white tw-cta-link font-weight-bold" id="partner-cta">
+                <a href="{{ localized_partner_page.url }}" class="text-white tw-cta-link font-weight-bold" id="partner-cta">
```

Comment:
Hi @danielfmiranda thank you for taking at look at this. Good observation! I just commited the changes with the `{% if %}` statement in order to avoid rendering when partner page is not set. Thank you!